ql/src/java/org/apache/hadoop/hive/ql/io/parquet/LeafFilterFactory.java

```diff
@@ -37,6 +37,9 @@
 
 public class LeafFilterFactory {
   private static final Logger LOG = LoggerFactory.getLogger(LeafFilterFactory.class);
+  public static final String MESSAGE = "Conversion to Parquet FilterPredicate not supported for %s "
```

Comment:
The error message is a bit clunky. ( Don't take it personal :) )
I would use the following error message. I think it is more expressive.  
_The conversion to Parquet FilterPredicate is not supported for %s. Please try to set the following configurations at the session level:  ...._

Reply:
Thank you @TuroczyX for reviewing. I have made the change as recommended.